antora-ui-camel/src/css/tabs.css

```diff
@@ -0,0 +1,98 @@
+/*! Asciidoctor Tabs | Copyright (c) 2018-present Dan Allen | MIT License */
```

Comment:
Let's use `@import '@asciidoctor/tabs/...` from above so we don't need to vendor this file, keeping it here makes it prone to being stale when we upgrade the @asciidoctor/tabs dependency